data/maps/ChiiTown_School/map.json

```diff
@@ -25,7 +25,7 @@
       "trainer_type": "TRAINER_TYPE_NONE",
       "trainer_sight_or_berry_tree_id": "0",
       "script": "ChiiTown_School_EventScript_ChildPokemonInsideVolcano",
-      "flag": "0"
+      "flag": "FLAG_TEMP_1"
```

Comment:
NPC kids in the Chii Town school still have temp flags set to them, is that normal?

Reply:
There was one commit with my changes that I couldn't undo; this might have been from that commit. But it should be okay; I'm planning to redo the schedule system anyway